src/main/java/frc/trigon/robot/constants/PathPlannerConstants.java

```diff
@@ -0,0 +1,72 @@
+package frc.trigon.robot.constants;
+
+import com.pathplanner.lib.auto.AutoBuilder;
+import com.pathplanner.lib.commands.PathfindingCommand;
+import com.pathplanner.lib.config.PIDConstants;
+import com.pathplanner.lib.config.RobotConfig;
+import com.pathplanner.lib.controllers.PPHolonomicDriveController;
+import com.pathplanner.lib.path.PathConstraints;
+import com.pathplanner.lib.pathfinding.Pathfinding;
+import frc.trigon.robot.RobotContainer;
+import org.json.simple.parser.ParseException;
+import org.trigon.hardware.RobotHardwareStats;
+import org.trigon.utilities.LocalADStarAK;
+import org.trigon.utilities.mirrorable.Mirrorable;
+
+import java.io.IOException;
+
+/**
+ * A class that contains the constants for everything related to PathPlanner.
```

Comment:
A class that contains the constants and configurations for everything related to PathPlanner